bentoml/bundler/loader.py

```diff
@@ -158,12 +158,14 @@ def load(bundle_path):
         with _resolve_remote_bundle_path(bundle_path) as local_bundle_path:
             return load(local_bundle_path)
 
-    track_load_start()
+    if not skip_artifact:
+        track_load_start()
```

Comment:
this and the `track_load_finish` below  are for usage tracking, not related to loading artifact

Reply:
I know. Loading service without artifact is different from the usage before. It take shorter time than g-unicorn workers obviously. I'm not sure if we could mix them up.